mkdocs.yml

```diff
@@ -102,21 +103,30 @@ nav:
   - Resources:
     - Cloud Resources: Resources/Cloud_Resources/index.md
     - FPGA Resource: Resources/FPGA_Resources/index.md
-    - GPU Resources: Resources/GPU_Resources/index.md
+    - GPU Resources:
+      - Overview: Resources/GPU_Resources/index.md
+      - CMS Resources:
+        - SWAN: Resources/GPU_Resources/CMS_Resources/SWAN.md
+        - LXPLUS Condor: Resources/GPU_Resources/CMS_Resources/lxplus_condor.md
+        - LXPLUS GPU: Resources/GPU_Resources/CMS_Resources/lxplus_gpu.md
+        - ml.cern.ch: Resources/GPU_Resources/CMS_Resources/ml_cern_ch.md
+        - Notebooks:
+          - PytorchMNIST: Resources/GPU_Resources/CMS_Resources/Notebooks/PytorchMNIST.md
+          - TopTaggingMLP: Resources/GPU_Resources/CMS_Resources/Notebooks/TopTaggingMLP.md
   - Tutorials:
```

Comment:
I think at this point this name should change from `Tutorials` to `Guides` or something similar.

Reply:
Ok, I can switch the naming, sure.